### PR TITLE
docs: Ruby tutorial

### PR DESCRIPTION
**Pull request by alvinometric**
A "getting started" for the Ruby SDK.

I used the Stars Wars API, in typical tutorial style.

### CODE REVIEW

**Review by sighphyre — APPROVED** [review]
Very cool! Left some thoughts but I don't see anything blocking

File: website/docs/feature-flag-tutorials/ruby/implementing-feature-flags-ruby.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+
+The Unleash Server is a **Feature Flag Control Service**, which manages your feature flags and lets you retrieve flag data. Unleash has a UI for creating and managing projects and feature flags. There are also [API commands available](https://docs.getunleash.io/reference/api/unleash) to perform the same actions straight from your CLI or server-side app.
+
+## 1. Best practices for backend apps with Unleash
+
+Ruby is a backend language, so there are special considerations to plan around when implementing feature flags.
+
+Most importantly, you must:
+
+- Limit feature flag payloads for scalability, security, and efficiency
```
[the remark this answers is not in the document]
Comment (nnennandukwe) [reply website/docs/feature-flag-tutorials/ruby/implementing-feature-flags-ruby.md:39]: I think this calls for a review by engineers of this section of text that we are using across our documentation.
We aim to be as accurate as possible with this so thanks for this feedback. I'll post about it in engineering rotation.

File: website/docs/feature-flag-tutorials/ruby/implementing-feature-flags-ruby.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+- Use graceful degradation where possible to improve the resiliency of your architecture
+
+A feature flagging system should not be the reason your app slows down or fails. That’s why we recommend limiting the size of your feature flag payloads. For example, instead of making one large call to retrieve flag statuses for all users, group your users by specific attributes as part of your targeting rules.
+
+Our SDKs cache your feature flag configuration to help reduce network round trips. You can rely on that cache if don’t have a Feature Flag Control Service, which will mitigate potential failure.
```
[the remark this answers is not in the document]
Comment (alvinometric) [reply website/docs/feature-flag-tutorials/ruby/implementing-feature-flags-ruby.md:44]: I'd be happy to remove that section altogether, what do you think @sighphyre ?

File: website/docs/feature-flag-tutorials/ruby/implementing-feature-flags-ruby.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+
+Run the code again:
+
+```ruby
+ruby main.rb
+```
+
+You should see `Hello GraphQL`, followed by `There are 60 planets` in your terminal.
+
+Yes, there are more planets with the GraphQL API. I’m not sure why but this a real-world scenario, with everything that comes with it.
```
[the remark this answers is not in the document]
Comment (alvinometric) [reply website/docs/feature-flag-tutorials/ruby/implementing-feature-flags-ruby.md]: I didn't know that, thank you 🙏

**Pull request merged**